reference/forms/types/submit.rst

```diff
@@ -34,6 +34,8 @@ useful when :doc:`a form has multiple submit buttons </form/multiple_buttons>`::
 Inherited Options
 -----------------
 
+Note some examples come from the ButtonType documentation. If you want to use the SubmitType field, you have to use SubmitType::class instead of ButtonType::class to generate a submit button.
```

Comment:
I would say:
"Note that some examples are inherited from the ButtonType documentation".

Also the use of `::class` looks weird here, maybe just replace `::` by a space?